src/Storage/Storage.Management/Az.Storage.psd1

```diff
@@ -53,7 +53,7 @@ DotNetFrameworkVersion = '4.7.2'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.1.2'; })
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.1.3'; })
```

Comment:
I don't think we need to upgrade Az.Accounts dependency here, since:
1. No Storage change depends on the new Az.Accounts change
2. Storage might release before new Az.Accounts release

Reply:
According to our discussion, we will upgrade Az.Accounts version for this OOB release.